### PR TITLE
Rename docker install and setup scripts

### PR DESCRIPTION
**Pull request by ximenesuk**
This PR renames the scripts used to build docker images. See https://github.com/ome/omero-install/issues/46 The one slight oddity remaining is that of `centos7` where there is just one `test` directory but two install scripts.

To test each variant should be built and run and a check made that the web server is available.

### CODE REVIEW

**Comment by ximenesuk** [comment]
This final commit changes the installation level `README.md`which was out of date due to the splitting of the scripts. It doesn't yet address Centos 7, I'll defer that to someone else.

**Comment by jburel** [comment]
closing this PR so we can create the various branches.

**Comment by jburel** [comment]
README:
should be `bash install_NAMEOFSCRIPT`

**Comment by jburel** [comment]
Adding an example of command on how to run will be useful. e.g. how to be able to access web/install

`docker run --rm -it -p 8080:80 -p 4063:4063 -p 4064:4064 omero_install_test_centos6_nginx`

**Comment by jburel** [comment]
centos6 both apache/nginx okay

**Comment by jburel** [comment]
General comment in dependencies files: we should run a clean post install e.g. `yum clean all`
Probably part of the scripts review. will add a card

**Comment by manics** [comment]
There's no reason to run `yum clean all` on a proper system

**Comment by jburel** [comment]
@manics: ok thanks but we can still clean things that are downloaded

**Comment by jburel** [comment]
ubuntu1404 nginx/apache both okay

**Comment by jburel** [comment]
Debian 8 failure: 

```
E: Failed to fetch http://httpredir.debian.org/debian/pool/main/j/javascript-common/javascript-common_11_all.deb  Error reading from server. Remote end closed connection [IP: 5.153.231.35 80
```

cc @dominikl

**Comment by jburel** [comment]
The following has been removed from the centos7-apache so it cannot be tested using the dev branch (necessary for apache testing)

```
if [ $OMEROVER = omerodev ]; then
    yum -y install python-virtualenv
fi
```

or `python-virtualenv` gets installed regardless of the branch used (with a comment!) in dependencies

**Comment by jburel** [comment]
centos7 both apache and nginx work, Need adjustment to support `omerodev` branch cf. https://github.com/ome/omero-install/pull/56#issuecomment-158415952

**Comment by jburel** [comment]
looking at the ubuntu dependencies file, `python-virtualenv` is installed so we should follow the same strategy for centos7 so the oremodev branch can be used.

**Comment by ximenesuk** [comment]
Sorry, the virtualenv install was killed accidentally with the intention of moving it somewhere else. I'll add to the dependencies and add an example to the README.

**Comment by jburel** [comment]
The debian issue has been introduced in this PR.
I do not have issue when I run `./docker-build.sh debian8` using develop

**Comment by manics** [comment]
You could drop the virtualenv completely and `pip install omego` globally in `*_omerodev.sh`

**Comment by ximenesuk** [comment]
@jburel that's odd as there has been no material change to the debian build in this PR, it still simply uses the ubuntu one.

**Comment by ximenesuk** [comment]
@manics you mean just drop the lines `virtualenv omego` in `step04_all_omerodev.sh` and `step04_centos6_py27_omerodev.sh`?

**Comment by manics** [comment]
@ximenesuk Yes

**Comment by ximenesuk** [comment]
Thanks for the feedback, I'll reconfigure the various files and investigate the debian issue.

**Comment by hflynn** [comment]
This numbered step naming scheme works for me, although it's a bit hard to follow the discussion over 2 different PRs.

**Comment by jburel** [comment]
@ximenesuk:could you re-open a PR so I can start using the new name format in the doc?
We can fix the issues in a follow-up PR
